migrations/20230609183358_update-friendship-history-migrated.up.sql

```diff
@@ -0,0 +1,32 @@
+
+UPDATE
+    friendship_history
+  SET
+    metadata = '{"migrated_from_synapse":  true}',
```

Comment:
QQ: is it possible for some rows to have existing metadata? In which case, would it be ok to override it?

Reply:
Regarding the empty metadata: All the `accept` events have no metadata as that was a decision made when migrating them.

And regarding the schema of the metadata column:

The other fields that are being used in the metadata field are:
- `message` -> that makes no sense for an `Accept event`
- `synapse_room_id` -> As it was migrated to the friendship table, there is no need to have it in the metadata anymore
In my opinion, if we add those fields that we already know their values will always remain null, doesn't add too much value to the table.

On the other side, having a JSON field in a table allows to have different schemas and that's the benefit of having it as a unique field instead of having a column for each key.

Reply:
Fair enough 👍🏻